nginx/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM phusion/baseimage
```

Comment:
@antero10 I think that just inheriting from the phusion/baseimage is not enough... You still need to use their init file.

https://github.com/phusion/baseimage-docker#getting-started